services/QuillLMS/app/controllers/api/v1/session_feedback_histories_controller.rb

```diff
@@ -0,0 +1,27 @@
+class Api::V1::SessionFeedbackHistoriesController < Api::ApiController
+  # GET /feedback_histories.json?page=1&activity_id=33
+  def index
+    options = {}
```

Comment:
Is this options left over from older code, not seeing it used.

Reply:
Yeah, good catch.  Removed.